### PR TITLE
Point becomes Vertex

### PR DESCRIPTION
**Pull request by codisart**
I renamed the PointInterface to VertexInterface for begin the renamming of Point Class to Vertex.

### CODE REVIEW

**Comment by codisart** [comment]
@toin0u 
I'm not sure that my last commit is smart.
I'm trying to not break everything but maybe we need to do so we don't miss all the places where it breaks. 
What do you think ?

**Comment by toin0u** [comment]
@codisart Thanks for the work :) I'm fine to break the API because this change is important actually! Could you also rename variable names as well?

**Comment by codisart** [comment]
Is the CLI/Command/Point a reference to the (ex) Point/Point class ?

**Comment by toin0u** [comment]
@codisart Yest it does - those commands uses the Point class.

**Comment by codisart** [comment]
I think i'm done.

File: src/CLI/Command/Vertex/FinalCardinal.php [hunk 1]
```diff
@@ -20,7 +20,7 @@
 use Symfony\Component\Console\Output\OutputInterface;
 
 /**
- * Command-line point:final-cardinal class
+ * Command-line povertexint:final-cardinal class
```
Comment (toin0u) [inline src/CLI/Command/Vertex/FinalCardinal.php]: I think it's just `vertex` here :)

**Comment by toin0u** [comment]
@codisart Thanks for your great work! I had just one comment. We can merge it afterwards :)

**Comment by codisart** [comment]
I did a rebase.

**Comment by toin0u** [comment]
@codisart Perfect! Thanks <3

**Pull request merged**